app/controllers/candidate_interface/sign_in_controller.rb

```diff
@@ -40,6 +34,26 @@ def create
       end
     end
 
+    def create_expired
+      if params[:u] && FeatureFlag.active?('improved_expired_token_flow')
+        begin
+          candidate_id = Encryptor.decrypt(params[:u])
+          @candidate = Candidate.find(candidate_id)
+        rescue ActiveSupport::MessageEncryptor::InvalidMessage
+          redirect_to candidate_interface_sign_in_path
+          return
+        end
+      end
+
+      if @candidate&.persisted?
```

Comment:
This will always be true, since `Candidate.find` would raise a RecordNotFound exception.